src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        return Color.values()[new Random().nextInt(Color.values().length)].name();
```

Comment:
1. Create "Random" as a class field. Don't create new Random every time when method getRandomColor() is invoked
2. Try not to make so difficult constructions. Your code must be easy to read
```suggestion
        int index = random.nextInt(Color.values().length);
        return Color.values()[index].name();
```